src/align.py

```diff
@@ -33,4 +72,43 @@ def edits(x: str, y: str) -> str:
     'MDMMMMMMIMMMM'
 
     """
-    return ""
+
+    # Better variable name, I think
+    align1 = x
+    align2 = y
+
+    # The two aligned files are saved at there variables
+    edits = ""
+
+    if len(align1) == len(align2):
+
+        for i in range(len(align1)):
+
+            if align1[i] != "-" and align2[i] != "-":
+                # If the bases are matches or mismatches, then M is added to edits
+                edits += "M"
+
+            elif align1[i] != "-" and align2[i] == "-":
+                # If there is a gap, "-", in align2, then a D is added
+                edits += "D"
+
+            elif align1[i] == "-" and align2[i] != "-":
```

Comment:
An `else` would suffice here. If you don't match any of the previous two, then you must match here.